### PR TITLE
Dockerhub

### LINKED ISSUE #78: Publish prediction service as a container on dockerhub
Currently someone who wants to get predictions from an Autofocus model has to download the whole repo, find the `autofocus/predict` module, and build a docker container before they can run it. Publishing the container itself would allow them to skip all of those steps.

### PR DESCRIPTION
**Pull request by gsganden**
Closes #78 

I pushed a docker image of the prediction serving app to Dockerhub. This PR adds instructions for using it to the repo README. Putting the app image on Dockerhub will make it easier for the Lincoln Park Zoo or anyone else simply to download and run the app without having to build it themselves.

I also simplified the app by removing `nginx`. My understanding is that `nginx` is helpful for handling web traffic, but it is unnecessary for local deployments, and it would complicate using Dockerhub because it involves running multiple containers. We can always put `nginx` back in later if we want to do a web deployment, but at this point I think local deployments work better because sending images over the internet would be a bottleneck.

Pull Request Checklist
 - [x] Pull request includes a description of the change and the reason behind it.
 - [x] Pull request [uses keywords to close relevant issues](https://help.github.com/en/articles/closing-issues-using-keywords).
 - [ ] Pull request includes unit tests for any new functionality.
 - [x] README and docs have been updated.
 - [x] `./.ci/test.sh` passes locally.

Maintainer's responsibilities:
- [x] `_version.py` has been updated.
- [x] `CHANGELOG.md` has been updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,62 @@ This project uses deep learning computer vision to label images taken by motion-
 - [Uptake.org Autofocus Case Study](https://www.uptake.org/impact/special-projects)
 - [Machine Learning Meets Wildlife Conservation](https://www.lpzoo.org/blog/machine-learning-meets-wildlife-conservation)
 
+## Getting the App
+
+If you just want to get labels for your images, you can use the following steps to run a service that passes images through a trained model.
+
+1. Make sure [Docker](https://www.docker.com/get-started) is installed and running.
+2. Run `docker pull gsganden/autofocus` to download the app image. (Note that it takes up about 4GB of disk space.)
```
Comment (jameslamb) [inline README.md]: this defaults to `gsganden/autofocus:latest`, is that what you want?

File: README.md [hunk 2]
```diff
@@ -11,13 +11,62 @@ This project uses deep learning computer vision to label images taken by motion-
...
+3. Run `docker run -p 8000:8000 gsganden/autofocus_serve` to start the app.
```
Comment (jameslamb) [inline README.md]: this name is inconsistent with the one you used above. You should pick whether it's `autofocus` or `autofocus_serve`

File: README.md [hunk 3]
```diff
@@ -11,13 +11,62 @@ This project uses deep learning computer vision to label images taken by motion-
...
+  "raccoon": 0.9986177682876587,
+  "rat": 4.3888848111350853e-10,
+  "skunk": 4.078452775502228e-07,
+  "squirrel": 1.2888597211713204e-06,
+  "unknown": 0.0004612557531800121,
+  "woodchuck": 1.2980818033154779e-14
+}
+```
+
+The `predict_zip` endpoint returns a JSON object mapping file paths to model probabilities formatted as above.
```
Comment (jameslamb) [inline README.md]: it's common to document endpoints with a leading `/`, so `/predict_zip`. This visually distinguishes them from functions or classes in your code

File: README.md (hunk 1 above)
Comment (gsganden) [reply README.md]: Sure? Would you suggest a different approach?

File: README.md (hunk 2 above)
Comment (gsganden) [reply README.md]: Good catch, fixed.

File: README.md (hunk 3 above)
Comment (gsganden) [reply README.md]: Fixed, thanks!

**Comment by gsganden** [comment]
@jameslamb I just updated `autofocus/predict/example_post.py` and `autofocus/predict/example_post.R`. Could you please take a look? Also, would you be able to make sure that `autofocus/predict/example_post.R` runs? I don't have an R environment set up.

File: README.md (hunk 1 above)
Comment (gsganden) [reply README.md]: I'm going to go ahead and merge, but I'd welcome more input on this point. Perhaps I should be explicit about version numbers?

**Pull request merged**